### DISCUSSION TITLE
Remove unnecessary .neoconf.json file

### DISCUSSION THREAD

**Discussion by dangdd2003** [LazyVim/LazyVim#3434] [General]
As i saw newest version of lazyvim has already moved neoconf.nvim to extras, so .neoconf.json file is no longer needed, right ? Or are there any other plugins that use that file ?

![image](https://github.com/LazyVim/LazyVim/assets/124034919/69d020fa-b367-47fd-b5f0-088cc7c26862)

**Comment by folke** [comment]
yep, you can remove it
